tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,26 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+from marionette_driver import By, Wait, expected
+
+class BugzillaLite(Base):
+
+    name = 'Bugzilla Lite'
+    _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
+    _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
+    _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _app_ready_locator = (By.XPATH, "//div[@class='icon']//span[contains(text(),'Bugzilla Lite')]")
```

Comment:
Actually I changed this part:         BugzLite_icon = self.marionette.find_element('xpath', "//div[@class='icon']//span[contains(text(),'Bugzilla Lite')]")
with this one _app_ready_locator = (By.XPATH, "//div[@class='icon']//span[contains(text(),'Bugzilla Lite')]")
to search for the icon App to launch it